app/src/main/res/layout/item_history_card_view.xml

```diff
@@ -0,0 +1,118 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="@dimen/history_card_view_width"
+    android:layout_height="@dimen/history_card_view_height"
+    android:layout_gravity="center|top"
+    android:background="@drawable/bg_history_cardview"
+    tools:ignore="ContentDescription,HardcodedText">
+
+    <ImageView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:src="@drawable/ic_card_view_rectangle"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <TextView
+        android:id="@+id/fromTV"
+        style="@style/Heading2"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/history_card_view_horizontal_margin"
+        android:layout_marginTop="@dimen/history_card_view_heading_margin_top"
+        android:text="@string/history_card_view_start_heading"
+        android:textColor="@color/history_card_view_heading"
+        android:textSize="@dimen/small_text_size"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <TextView
+        android:id="@+id/startLocationTV"
+        style="@style/Heading1"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/history_card_view_horizontal_margin"
+        android:text="Strada Lunii 2A"
```

Comment:
I assume you wanted to use `tools` here for preview? This applies in a couple other places as well